SimPEG/Mesh/DiffOperators.py

```diff
@@ -401,14 +415,19 @@ def fget(self):
         return locals()
     cellGrady = property(**cellGrady())
 
+    def _cellGradzStencil(self):
+        if self.dim < 3: return None
+        BC = ['neumann', 'neumann']
```

Comment:
this is cut and pasted from inside cellGradz (previously lines 409-411). Should this be calling `self.setCellGradBC(self._cellGradBC_list)`